sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/EliminateSortsSuite.scala

```diff
@@ -308,4 +308,27 @@ class EliminateSortsSuite extends PlanTest {
     val correctAnswer = PushDownOptimizer.execute(noOrderByPlan.analyze)
     comparePlans(optimized, correctAnswer)
   }
+
+  test("should not remove orderBy in sortBy clause") {
```

Comment:
@apapi . Please change like the following.
```scala
- test("should not remove orderBy in sortBy clause") {
+ test("SPARK-30408 should not remove orderBy in sortBy clause") {
```

Reply:
Done.